runtime/src/lib.rs

```diff
@@ -30,6 +30,7 @@ pub mod mocks;
 cfg_if::cfg_if! {
     if #[cfg(not(target_arch = "wasm32"))] {
         pub mod tokio;
+        mod storage;
```

Comment:
Once we have more in `storage`, I think it would be worth making this `pub`

Reply:
In preparation for that, maybe worth putting a `//!` comment at the top of storage/mod.rs

Reply:
Added a one-liner at the top